encrypt-main.jsx

```diff
@@ -19,7 +19,7 @@ function redirect(state, replace) {
   if (state.location.query.video) {
     replace(`/encrypt/${pageType}/${state.location.query.video}`);
   } else {
-  	replace(`/encrypt/${pageType}/${encryptVideos.length}`);
+  	replace(`/encrypt/${pageType}/${encryptVideos.length}${state.location.search}`);
```

Comment:
I think it is better to do it this way? https://github.com/reactjs/react-router/blob/master/docs/API.md#replacepathorloc